### ISSUE TITLE
Problems debugging on MacOS-Catalina

### ISSUE THREAD

**Issue by andreeis** [microsoft/vscode-cmake-tools#1198]
Opening a new issue for the last comment in https://github.com/microsoft/vscode-cmake-tools/pull/988 (from @coppersoft)

I'm sorry, but this workaround didn't work for me.

OS: MacOS X Catalina: 10.15.4
Visual Studio Code: 1.44.2
ms-vscode.cpptools: 0.27.0
ms-vscode.cmake-tools: 1.3.1

I downloaded Xcode 10.3, extracted llvm-mi from it and put it inside

/Applications/Xcode.app/Contents/Developer/usr/bin/

Then I added this to settings.json

"cmake.debugConfig": {
        "miDebuggerPath": "/Applications/Xcode.app/Contents/Developer/usr/bin/lldb-mi"
}
But nothing changes. I launch the debug, but everything hangs, no breakpoint hit and it seems that nothing is executed.

I tried also setting "miDebuggerPath" to the llvm-mi bundled in the cpptools

~/.vscode/extensions/ms-vscode.cpptools-0.27.0/debugAdapters/llvm-mi

But nothing changes...

Am I doing something wrong?

Thank you

**Comment by wtlgo** [comment]
Same problem here

- macOS Mojave, Version: 10.14.6
- Visual Studio Code, Version: 1.44.2
- CMake Tools, Version: 1.3.1
- cpptools, Version: 0.27.0

My settings.json. I built `lldb-mi` from source.
```json
{
    "cmake.debugConfig": {
        "miDebuggerPath": "/usr/local/bin/lldb-mi"
    }
}
```

And output output when I click "Debug"
```shell
[proc] Executing command: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/lldb --version
```

I don't know what am I doing wrong.

**Comment by wtlgo** [comment]
I checked and turned out that I was doing nothing wrong, it was just a bug in v1.3.1. Seems like this bug was already fixed in 70e935ba8acf5fcf16f1f46dc497a943e231dd22.

**Comment by andreeis** [comment]
Oh, that is great. Thank you for digging through our last months commits and for identifying the fix for this problem. It will be released with CMake Tools 1.4.

**Comment by bobbrow** [comment]
Duplicate of #1060
